examples/analytics.amp.html

```diff
@@ -75,6 +75,8 @@
 </script>
 </amp-analytics>
 
+<amp-analytics id="analytics3" config="analytics.config.json"></amp-analytics>
```

Comment:
Make the URL `./analytics.config.json` to make it more obvious what is happening here.